### PR TITLE
Db

### PR DESCRIPTION
**Pull request by markus0059**

### CODE REVIEW

**Review by Guillaum6e — APPROVED** [review]

File: database.sql [hunk 1]
```diff
@@ -1,48 +1,286 @@
...
-  photo VARCHAR(255) NOT NULL,
-  rating INT DEFAULT 5
-);
+CREATE TABLE
+    user (
+        id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+        pseudo VARCHAR(20) NOT NULL,
+        lastname VARCHAR(20) NOT NULL,
+        firstname VARCHAR(20) NOT NULL,
+        adress VARCHAR(80) NOT NULL,
```
Comment (Benja-san) [inline database.sql:9]: address

File: database.sql [hunk 2]
```diff
@@ -1,48 +1,286 @@
...
+CREATE TABLE
+    product (
+        id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+        title VARCHAR(20) NOT NULL,
+        price INT NOT NULL,
+        description TEXT NOT NULL,
+        photo VARCHAR(255) NOT NULL,
+        status VARCHAR(50) DEFAULT 'en vente',
+        material VARCHAR(50),
+        category_item VARCHAR(50),
```
Comment (Benja-san) [inline database.sql]: category_id with many to one

File: database.sql [hunk 3]
```diff
@@ -1,48 +1,286 @@
...
-);
+CREATE TABLE
+    product (
+        id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+        title VARCHAR(20) NOT NULL,
+        price INT NOT NULL,
+        description TEXT NOT NULL,
+        photo VARCHAR(255) NOT NULL,
+        status VARCHAR(50) DEFAULT 'en vente',
+        material VARCHAR(50),
```
Comment (Benja-san) [inline database.sql]: Choose between many to many relationship OR change for JSON type

File: database.sql [hunk 4]
```diff
@@ -1,48 +1,286 @@
...
+        category_item VARCHAR(50),
+        category_room VARCHAR(50),
```
Comment (Benja-san) [inline database.sql]: JSON or many to many

File: database.sql [hunk 5]
```diff
@@ -1,48 +1,286 @@
...
+        color VARCHAR(50),
```
Comment (Benja-san) [inline database.sql]: Json or many to many

File: database.sql [hunk 6]
```diff
@@ -1,48 +1,286 @@
...
+        `user_id`,
+        `description`
+    )
+VALUES (
+        'chaise en bois',
+        50,
+        'https://images.pexels.com/photos/116910/pexels-photo-116910.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1',
+        'bois',
+        'ameublement',
+        '["bureau","salle à manger"]',
```
Comment (Benja-san) [inline database.sql:148]: {"bureau", "salle à mager"} or {"title" : "bureau", "title" : "salle à manger"}

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**